app/controllers/distributions_controller.rb

```diff
@@ -96,6 +96,12 @@ def new
   def show
     @distribution = Distribution.includes(:line_items).includes(:storage_location).find(params[:id])
     @line_items = @distribution.line_items
+
+    @total_quantity = @line_items.sum(&:quantity)
```

Comment:
Since this method is being used in two places now, what do you think about adding an instance method on `Distribution` to just spit out that number?

Reply:
I added `Itemizable#total_quantity` in the 3rd commit. Does that commit make sense?